modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
```

Comment:
How does the user choose a watchdog action? Select from a list? Select a check box? Please see IBM style guide for the appropriate verb. I can't tell whether this is a UI or not. If it is a UI, it's a good idea to start with "In the OpenShift console, ..." so that the user has a context for this step.

Reply:
On second thought, it looks like these are options for the user to put in the yaml. So mentally selecting an action is not really a procedure step. I recommend adding them as callouts to the yaml.